components/ClaimPage/ConnetWallet.tsx

```diff
@@ -82,14 +82,17 @@ const ConnectWallet = ({ handleClose }: ConnectWalletProps) => {
         Connect
       </div>
       <WalletConnectButton>
-        <Button
-          id="wallet_connect_btn_in_redeem"
-          className="!uppercase w-[270px] 
+        {({ openConnectModal }) => (
+          <Button
+            id="wallet_connect_btn_in_redeem"
+            className="!uppercase w-[270px] 
           !bg-[white] !text-black dark:!bg-[black] dark:!text-[white]
           !shadow-[0px_4px_4px_rgb(255,255,255,0.25)] dark:!shadow-[0px_4px_4px_rgb(0,0,0,0.25)]"
-        >
-          Connect
-        </Button>
+            onClick={openConnectModal}
+          >
+            Connect
+          </Button>
+        )}
```

Comment:
why are you updating `ClaimPage` components?

is this component being used on `/Mint` page?

should this component be moved to a `shared` components directory?

Reply:
while integrating connect wallet button.
I recognized some unexpected integration.
So, I have updated WalletConnectButton.tsx component a bit.
![image](https://github.com/defientco/CRE8ORS-UI/assets/127037083/6510244b-fafc-48c8-8247-6bb0f538801a)
like this.
If we use some div or the other html tags as children, I think it can be make some warning on html formatting.
Because div can be inside button.
So, I thought this is better solution for that.